### PR TITLE
reformatted to improve readability of README

### PR DESCRIPTION
**Pull request by ichikuma7**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,24 +2,32 @@
...
+
 MO2 was not completed and was left with many issues. LePresidente managed to fix it to a usable state by himself after SkyrimSE came out, adding support for it, and releasing v2.0.8.
+
 The project took up speed again after a few more coders showed up in late 2017, and a lot of progress has been made.
 
 ## Help Wanted!
+
 Mod Organizer 2 is an open project in the hands of the community, there are problems that need to be solved and things that could be added. MO2 really needs developers and if you have the programming skills and some free time you can really improve the experience of the modding community.
 
 To have more information, please join the open MO2 Development discord server :* [ModOrganizerDevs](https://discord.gg/vD2ZbfX)
```
Comment (SuperSandro2000) [inline readme.md]: The star after : is a typo I guess.

**Comment by SuperSandro2000** [comment]
And I don't think that a paragraph new paragraph should start after each sentence. A middle ground between now and this PR would be ideal.

**Comment by isanae** [comment]
This adds way too many newlines, basically per sentence.

**Comment by Al12rs** [comment]
Closing since no one seems to agree on the proposed change.

**Pull request closed without merging**